test/sequential/test-benchmark-http.js

```diff
@@ -20,6 +20,9 @@ const path = require('path');
 
 const runjs = path.join(__dirname, '..', '..', 'benchmark', 'run.js');
 
+const env = process.env;
```

Comment:
at least `const env = Object.assign({}, process.env)` makes it explicit that you are using a copy, and not mutating the current process's env